frontend/tests/controllers/UserIdentityAssociationTest.php

```diff
@@ -45,28 +51,48 @@ public function testAssociateIdentityWithUser() {
         $user = UserFactory::createUser();
         $login = self::login($user);
 
-        $associatedIdentities = UserController::apiListAssociatedIdentities(new Request([
-            'auth_token' => $login->auth_token,
-        ]));
+        $associatedIdentities = UserController::apiListAssociatedIdentities(
+            new Request(['auth_token' => $login->auth_token,])
```

Comment:
este cambio es necesario? si sí, al menos hay que quitar esa coma innecesaria.

```suggestion
            new Request(['auth_token' => $login->auth_token])
```

igual en el resto de los lugares donde hay una coma antes de corchete en la misma línea.

Reply:
No, se puede quedar como estaba.